include/certifier_framework.h

```diff
@@ -361,6 +369,13 @@ class secure_authenticated_channel {
                        key_message & private_key,
                        const string &private_key_cert);
 
+  bool init_client_ssl(const string &          host_name,
+                       int                     port,
+                       const cc_trust_manager &mgr);
+  bool init_server_ssl(const string &          host_name,
+                       int                     port,
+                       const cc_trust_manager &mgr);
+
   void server_channel_accept_and_auth(
       void (*func)(secure_authenticated_channel &));
 
```

Comment:
Didn't we agree to refactor the read/write interfaces?

Reply:
I did except for readmsg and writemsg where I think there are problems.  I'd do that in a subsequent pull if we decide to proceed.